common/data_refinery_common/migrations/0020_update_qn_bucket.py

```diff
@@ -18,17 +19,25 @@ def update_qn_bucket(apps, schema_editor):
     We can't import the ComputedFile model directly as it may be a newer
     version than this migration expects. We use the historical version.
     '''
+
+    if not settings.RUNNING_IN_CLOUD:
```

Comment:
Can we confirm this is set correctly, even it's run on the deploy box? Does that count as "the cloud"?

Reply:
Yes, when this is run on the deploy box it's done so using whatever the environment that is being deployed is: https://github.com/AlexsLemonade/refinebio/blob/dev/infrastructure/deploy.sh#L252